spec/person_spec.rb

```diff
@@ -0,0 +1,67 @@
+require_relative '../person'
+require_relative '../book'
+require_relative '../rental'
+describe Person do
+  before :each do
+    @person1 = Person.new(12, name: 'Little Smith', parent_permission: false)
+    @person2 = Person.new(500, name: 'Elrond HalfElf')
+    @book1 = Book.new('Game of Thrones', 'George Martin')
+    @book2 = Book.new('The Songs of Achilles', 'Madelaine Miller')
+    @book3 = Book.new('The Lord of the Rings', 'Tolkien')
+    @rental = Rental.new(Time.now, @person1, @book1)
+  end
+  context 'When creating a new Person' do
+    it 'should have the following attributes (id, age, name, and parent_permission, rentals)' do
```

Comment:
**You did a great job! I like all the implementations!!!** 

- [ ] However, we have the last test that needs to be written. You need to write a test for the name if we do not provide a name it should return us the default value "Unknown". Kindly add a test for that. Thank you!